cdap-ui/app/cdap/components/FieldLevelLineage/v2/Context/FllContext.tsx

```diff
@@ -140,6 +163,64 @@ export class Provider extends React.Component<{ children }, IContextState> {
     });
   };
 
+  private fetchFieldLineage(qParams, timeParams, dataset = this.state.target) {
+    const namespace = getCurrentNamespace();
+    const updateState = (newState) => this.setState(newState);
+    getFieldLineage(namespace, dataset, qParams, timeParams, updateState);
+  }
+
+  private updateLineageFromRange(selection: string, start: number | string, end: number | string) {
```

Comment:
Can use the same custom type that you define. This is missing `null` which is really not the type of `start` and `end` that you started with.